docs/reference/settings/audit-settings.asciidoc

```diff
@@ -57,17 +64,26 @@ audited in plain text when including the request body in audit events.
 [[node-audit-settings]]
 ==== Local Node Info Settings
 
-`xpack.security.audit.logfile.prefix.emit_node_name`::
-Specifies whether to include the node's name in the local node info. The
-default value is `true`.
-
-`xpack.security.audit.logfile.prefix.emit_node_host_address`::
-Specifies whether to include the node's IP address in the local node info. The
-default value is `false`.
-
-`xpack.security.audit.logfile.prefix.emit_node_host_name`::
-Specifies whether to include the node's host name in the local node info. The
-default value is `false`.
+`xpack.security.audit.logfile.emit_node_name`::
+Specifies whether to include the <<node-name, node name>> as a field in each
```

Comment:
This link fails when I build the documentation.  Are you trying to link to https://www.elastic.co/guide/en/elasticsearch/reference/master/node.name.html ?

Reply:
yes.

Reply:
replaced with `{ref}/node.name.html[node name]`